app/safe/internal/state/io-decrypt.go

```diff
@@ -57,5 +87,9 @@ func decryptDataFromDisk(key string) ([]byte, error) {
 		return nil, errors.Wrap(err, "decryptDataFromDisk: Error reading file")
 	}
 
+	if env.SafeFipsCompliant() {
+		return decryptBytesAes(data)
+	}
+
```

Comment:
An alternative would be using an interface/plugin to let people integrate their encryption algorithms. — That’s an overkill for now, but we may want to implement it later.